net/wget2/Portfile

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                wget2
+version             2.1.0
+revision            0
+
+homepage            https://gitlab.com/gnuwget/wget2
+
+description         \
+    GNU Wget2 is the successor of GNU Wget, a file and recursive website \
+    downloader.
+
+long_description    {*}${description}.
+
+categories          net www
+installs_libs       no
+license             GPL-3
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+master_sites        https://ftp.gnu.org/gnu/wget
+
+checksums           rmd160  3e39a932a90eaaff69e90ecb761a4946e3444e10 \
+                    sha256  a05dc5191c6bad9313fd6db2777a78f5527ba4774f665d5d69f5a7461b49e2e7 \
+                    size    3867884
+
+depends_build-append \
+                    port:doxygen \
+                    port:pkgconfig
+
+depends_lib-append  path:lib/pkgconfig/gnutls.pc:gnutls \
+                    port:brotli \
+                    port:bzip2 \
+                    port:gpgme \
+                    port:libhsts \
+                    port:libidn2 \
+                    port:libmicrohttpd \
+                    port:libpsl \
+                    port:lzma \
+                    port:nghttp2 \
+                    port:pcre2 \
+                    port:zlib \
+                    port:zstd
+
+configure.args-append \
+                    --disable-silent-rules \
+                    --with-plugin-support
+
+configure.args-append \
+                    --with-brotlidec \
+                    --with-bzip2 \
+                    --with-gpgme \
+                    --with-libhsts \
+                    --with-libidn2 \
+                    --with-libmicrohttpd \
+                    --with-libnghttp2 \
+                    --with-libpcre2 \
+                    --with-libpsl \
+                    --with-lzma \
+                    --with-ssl=gnutls \
+                    --with-zlib \
+                    --with-zstd
+
+post-destroot {
+    # Install man page for wget2. (This is normally installed automatically
+    # if using pandoc.)
```

Comment:
I see that you don't depend on pandoc. But are you doing anything to ensure that pandoc doesn't get used if it happens to be installed?

Reply:
Unfortunately, the way this `./configure` script is written, there is just one flag that disables _all_ doc generation, both via `doxygen` (library docs) and `pandoc` (man page).  Still looking for a reliable way to do this without having to patch `./configure` directly...